ewms_pilot/config.py

```diff
@@ -0,0 +1,41 @@
+"""Configuration constants."""
+
+import dataclasses as dc
+from typing import Optional
+
+from wipac_dev_tools import from_environment_as_dataclass
+
+# pylint:disable=invalid-name
+
+#
+# Env var constants: set as constants & typecast
+#
+
+
+@dc.dataclass(frozen=True)
+class EnvConfig:
+    """For storing environment variables, typed."""
+
+    # broker -- assumes one broker is the norm
+    EWMS_PILOT_BROKER_CLIENT: str = "pulsar"
+    EWMS_PILOT_BROKER_ADDRESS: str = "localhost"
+    EWMS_PILOT_BROKER_AUTH_TOKEN: str = ""
+
+    # queues
+    EWMS_PILOT_QUEUE_INCOMING: str = ""
+    EWMS_PILOT_QUEUE_OUTGOING: str = ""
```

Comment:
My concern with this is that not all workflows will have a single queue in and a single queue out.  Particularly, IceCube simprod will have more (especially outgoing).

Reply:
Do you mean there would be multiple pilots (with different in and out queue) or that the pilot itself will need an enhancement to allow multiple in/out queues?

Reply:
The pilot itself will have multiple in/out queues.  Probably more likely multiple out rather than in (routing based on some factor, like estimated memory usage).

Reply:
roger that. I'll revert those